cmd/bosun/expr/expr.go

```diff
@@ -238,6 +240,10 @@ func (r ResultSlice) Len() int           { return len(r) }
 func (r ResultSlice) Swap(i, j int)      { r[i], r[j] = r[j], r[i] }
 func (r ResultSlice) Less(i, j int) bool { return r[i].Value.(Number) > r[j].Value.(Number) }
```

Comment:
So you want me to make the ByValue type, get rid of this, and replace any sort calls on ResultSlice to cast to ByValue and then sort?